### ISSUE TITLE
Need to create many maps

### ISSUE THREAD

**Issue by Kiwiggle** [lriaudel/cpt-to-map-store#3]
Hi,

(I'm french but here I guess we have to speak english, so..)

First of all, I love your plugin that works fine on my project. Thank you.
But I need to create 2/3 maps  with your plugin. How could I do it ? Duplicating the plugin ? 
I'm not used with using WordPress, I have to use it for work purposes and I have never created any plugin so far. 
It would help me a lot. I can help, I know how to use PHP if there is a need. :)

**Comment by lriaudel** [comment]
Hi,

Thanks a lot. 
I started to evolve the dev to have a single card that can be called by an id. But apparently, I'm not finished...

**Comment by lriaudel** [comment]
Can you test the plugin with my last modification ?
https://github.com/lriaudel/cpt-to-map-store/archive/master.zip

**Comment by lriaudel** [comment]
![image](https://user-images.githubusercontent.com/4172532/89730056-9e2e9500-da3b-11ea-8e0d-b58f80a03fe3.png)

**Comment by lriaudel** [comment]
Oh no, I forgot for many map to display...

**Comment by Kiwiggle** [comment]
Yes, it works fine, I have no problem charging the map, I just need to create many maps. :)
I can't tell you until next tuesday when I'll come back to work to test it. 
Thanks for your reply.

**Comment by lriaudel** [comment]
All right, I just made a few more changes.
You can now test Tuesday. ;-)

**Comment by Kiwiggle** [comment]
Hi, I've just tested what you added.
But I guess I have not explained that well. lol
I need to create many maps from many custom posts.

I have customs posts Résidences and Stationnements, and I need to create a map for Résidences AND for Stationnements.
Now we can create only one map, for only one custom post... But the idea with the id is good actually, even if I can't use it now

I hope it's better, I'm sorry

**Comment by lriaudel** [comment]
Actually, it wasn't clear.
I don't have time to explore it right now,, but you can propose a pull request.
